ansible/inventory/inventory

```diff
@@ -0,0 +1,20 @@
+[microshift]
+microshift.example.com ansible_host= private_ip= # microshift node
+
+[microshift:vars]
+ansible_user=centos
+ansible_become=yes
+pbench_key_url=
+pbench_config_url=
+
+[logging]
+logging.example.com ansible_host= private_ip= # logging node
+
+[logging:vars]
+ansible_user=centos
```

Comment:
Can we use `microshift` user here?

Reply:
Same as my previous comment for the `microshift:vars` for `ansible_user`.